rules/windows/defense_evasion_posh_compressed.toml

```
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2021/10/19"
+maturity = "production"
+updated_date = "2021/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of decompression and base64 decoding combined in PowerShell scripts, which Malware and Hacktools
```

```suggestion
Identifies the use of .Net functionality for decompression and base64 decoding combined in PowerShell scripts, which Malware and security tools
```

will need to re-lint this for length

---

I think there are several other rules which rely on calls to .Net classes and functionality which we do not mention in the description which could use similar language.